packages/core/src/components/ModalNew/ModalFooter/ModalFooterBase/ModalFooterBase.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { forwardRef } from "react";
+import styles from "./ModalFooterBase.module.scss";
+import Button from "../../../Button/Button";
+import Flex from "../../../Flex/Flex";
+import { ModalFooterBaseProps } from "./ModalFooterBase.types";
+import cx from "classnames";
+
+const ModalFooterBase = forwardRef(
+  (
+    { primaryButton, secondaryButton, renderAction, id, className, "data-testid": dataTestId }: ModalFooterBaseProps,
+    ref: React.ForwardedRef<HTMLDivElement>
+  ) => {
+    return (
+      <Flex
+        ref={ref}
+        id={id}
+        justify={Flex.justify.SPACE_BETWEEN}
+        gap={Flex.gaps.SMALL}
+        className={cx(styles.footer, className)}
+        data-testid={dataTestId}
+      >
+        <Button onClick={primaryButton.onClick} className={primaryButton.className}>
+          {primaryButton.text}
+        </Button>
+        {secondaryButton && (
+          <Button kind={Button.kinds.TERTIARY} className={secondaryButton.className} onClick={secondaryButton.onClick}>
+            {secondaryButton.text}
+          </Button>
+        )}
+        {renderAction}
```

Comment:
Wondering if it should be `children`

Reply:
we're rethinking everything here, but yep it can definitely be children

Reply:
ok we decided to leave everything here as slots
I want to keep the "children" prop as a safe prop for an option to pass a whole different footer which gets the footer css but can pass a custom one
I'm not sure this use case would ever happen, but you can never know